docs/storage.rst

```diff
@@ -146,6 +146,129 @@ start to diminish.
 
   - **Vacuum Interval**: The interval in hours between running a database vacuum command to reclaim space. Setting this too high will impact performance, setting it too low will mean that more storage may be required for longer periods.
 
+Using a Remote PostgreSQL Server
+--------------------------------
+
+Follow the steps below to set up PostgreSQL on a remote machine and enable remote connections securely.
+
+1. Install PostgreSQL on the Remote Machine
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Refer to the section :ref:`postgres-installation` for detailed PostgreSQL installation instructions.
```

Comment:
please fix the code of reference to postgres-installation:
![image](https://github.com/user-attachments/assets/d2d20a27-b137-4124-9c7d-4de7fe341834)